dv/uvm/core_ibex/Makefile

```diff
@@ -1,12 +1,15 @@
-# Copyright lowRISC contributors.
-# Licensed under the Apache License, Version 2.0, see LICENSE for details.
-# SPDX-License-Identifier: Apache-2.0
+# Top-Level Makefile
+###############################################################################
 
 .SUFFIXES:
+export
 
 # Explicitly ask for the bash shell
 SHELL := bash
 
+# Build the 'all' target by default, override with e.g. GOAL=rtl_tb_compile
+GOAL ?= all
```

Comment:
Is this correct? We have no 'all' target in this makefile but we do have a 'run'

Reply:
In run target, its calling,
```
@$(MAKE) --file wrapper.mk --environment-overrides --no-print-directory $(GOAL)
```
and wrapper.mk has 'all' target, does it make sense or i am missing something? @hcallahan-lowrisc

Reply:
Yes that's correct, it just passes the GOAL variable onto the wrapper.mk file, which does have an 'all'target